src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -1304,7 +1304,7 @@ public String getFeedbackSessionEndTimeValue() {
     }
 
     public InstructorFeedbackSessionsPage deleteSession() {
-        clickAndConfirm(getDeleteSessionLink());
+        click(getDeleteSessionLink());
```

Comment:
I think it's still reasonable to have a `clickAndConfirm` for soft-deletes.

Reply:
Explained as above :)